Symfony/CS/Fixer/Contrib/LogicalNotOperatorsWithSpacesFixer.php

```diff
@@ -28,15 +27,13 @@ public function fix(\SplFileInfo $file, $content)
         $tokens = Tokens::fromCode($content);
 
         for ($index = $tokens->count() - 1; $index >= 0; --$index) {
-            $token = $tokens[$index];
-
-            if ($tokens->isUnaryPredecessorOperator($index) && $token->equals('!')) {
-                if (!$tokens[$index + 1]->isWhitespace()) {
-                    $tokens->insertAt($index + 1, new Token(array(T_WHITESPACE, ' ')));
+            if ($tokens->isUnaryPredecessorOperator($index) && $tokens[$index]->equals('!')) {
```

Comment:
please swap the two checks, `equals` is faster than `isUnaryPredecessorOperator`.
Or remove the `isUnaryPredecessorOperator` check completely because `!` is always a unary predecessor operator.